row_batch.go

```diff
@@ -23,6 +24,19 @@ func (e *RowBatch) Values() []RowData {
 	return e.values
 }
 
+func (e *RowBatch) EstimateByteSize() uint64 {
+	var total int
+	for _, v := range e.values {
+		size, err := json.Marshal(v)
+		if err != nil {
+			continue
+		}
+		total += len(size)
```

Comment:
This will be an overestimate, no? Any `int64` nominally will consume 8 bytes but if turned into a string can be a lot higher, no?

Reply:
Also how will this interact with binary data stored in `blob` types?

Reply:
Just checked with `MEDIUMBLOB` and a txt file read as binary data locally and it works fine. Yes there's some overestimate but i think since we're "consistently" overestimating it should be fine?